### PR TITLE
[CI] Add dneto0 spirv-samples to lazy task

### PR DESCRIPTION
**Pull request by Gordon-F**
Test run - https://github.com/Gordon-F/naga/runs/2943811724?check_suite_focus=true

Part of #747

### CODE REVIEW

File: .github/workflows/lazy.yml [hunk 1]
```diff
@@ -76,3 +76,77 @@ jobs:
...
+          profile: minimal
+          toolchain: stable
+      - name: Download shaders
+        run: git clone https://github.com/dneto0/spirv-samples.git
+      - name: Build Naga
+        run: cargo build --release --bin naga
+      - name: Install vulkan-sdk
+        run: |
+           wget -qO - https://packages.lunarg.com/lunarg-signing-key-pub.asc | sudo apt-key add -
+           sudo wget -qO /etc/apt/sources.list.d/lunarg-vulkan-1.2.176-focal.list https://packages.lunarg.com/vulkan/1.2.176/lunarg-vulkan-1.2.176-focal.list
```
Comment (kvark) [inline .github/workflows/lazy.yml:95]: is this necessary? can we just install `vulkan-tools` from the standard package repositories?

File: .github/workflows/lazy.yml [hunk 2]
```diff
@@ -76,3 +76,77 @@ jobs:
...
+           sudo apt install vulkan-sdk
+      - name: Compile spv from spvasm
+        run: |
+           cd spirv-samples
+           mkdir -p spv
+
+           find "./spvasm" -name '*.spvasm' | while read fname;
+           do
+               echo "Convert to spv with glslc: $fname"
+               glslc --target-spv=spv1.3 $(realpath ${fname}) -o ./spv/$(basename ${fname}).spv
```
Comment (kvark) [inline .github/workflows/lazy.yml:106]: we should use `spirv-as` instead of `glslc` for this

**Review by kvark — COMMENTED** [review]
Very nice, thank you!
Just a few notes

File: .github/workflows/lazy.yml (hunk 2 above)
Comment (Gordon-F) [reply .github/workflows/lazy.yml:106]: I don't know how I can fix this error (Everything is ok on my machine 😄):
```
Convert to spv with glslc: ./spvasm/SpvParserImportTest_Import_NonSemantic_IgnoredExtInsts.spvasm
error: 26: 27: Couldn't translate unknown extended instruction name 'ArgumentInfo' to unsigned integer.
```

Looks like `spirv-tools` is outdated on build agent:
```
SPIRV-Tools v2020.1 unknown hash, 2020-03-23T08:58:00
Target: SPIR-V 1.5
```

https://github.com/Gordon-F/naga/runs/2953885438?check_suite_focus=true

Comment (kvark) [reply .github/workflows/lazy.yml:106]: does it matter that spirv-tools is from 2020? it should still work. It's not like we are using any fancy features in SPIR-V anyway

Comment (Gordon-F) [reply .github/workflows/lazy.yml]: It works with latest prebuild [spirv-tools](https://github.com/KhronosGroup/SPIRV-Tools).

https://github.com/Gordon-F/naga/runs/2954042540?check_suite_focus=true

**Review by kvark — APPROVED** [review]
Awesome work, thank you!

**Pull request merged**